subsys/bluetooth/host/hci_core.c

```diff
@@ -83,6 +83,10 @@ static bt_ready_cb_t ready_cb;
 
 static bt_le_scan_cb_t *scan_dev_found_cb;
 
+#if defined(CONFIG_BT_HCI_VENDOR_EVENT_CALLBACK)
+static bt_hci_vnd_evt_cb_t *hci_vnd_evt_cb;
```

Comment:
I don't think we use `*` anywhere for function pointers. You also don't use it e.g. in the `bt_hci_register_vnd_evt_cb` parameter type so just drop it from here as well.

Reply:
We use right above in scan_dev_found_cb

Reply:
right (sorry for being blind) - still seems inconsistent to me. I need to take a look at the wider code base since we should probably just make all these consistent with each other.

Reply:
These actually differ in the type definition as well. Some have `typedef int (*foo)(args...);` whereas others have `typedef int foo(args...);`. Maybe that determines the necessity for the `*` in the variable declarations (my C-knowledge is a bit lacking on this aspect of function pointer syntax). Either way, I suppose we want to make this consistent throughout the code base (which is a topic for a separate PR).

Reply:
I guess it depends if you wan't a typedef the refers to a function or to a pointer to a function. For me personally it's a bit clearer when the variable is a pointer to a function.